Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Script_file_added_to_startup-related_Registry_keys.yml

```diff
@@ -0,0 +1,2980 @@
+id: silent-Script file added to startup-related Registry keys
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: silent-Script file added to startup-related Registry keys
+issilent: true
+description: |-
+  This playbook is designed to handle "Script file added to startup-related Registry keys" alerts.
+
+  Playbook Stages:
+
+  Analysis:
+  - Extract the script path from the registry key value.
+  - Check the reputation of the associated processes.
+
+  Investigation:
+
+  - Searches for related XSIAM alerts to identify potential attack patterns.
+  - Retrieve the script file and relevant details for investigation.
+  - Analyze the script for malicious or suspicious parameters:
+    - If malicious parameters are found, proceed to remediation.
+    - If suspicious parameters are found, detonate and analyze the script using WildFire Sandbox.
+  - Checks the script file reputation.
+  - Detonate and analyze the script using WildFire Sandbox.
+
+  Remediation:
+
```

Comment:
- Terminates the malicious process.
  - Quarantines the malicious process.
  - Adds the malicious process hash to the blocklist.
  - Calculates the script file’s SHA256 hash if not already available.
  - Deletes the Registry key value (requires analyst approval).
  - Adds the script file hash to the blocklist (requires analyst approval).
  - Automatically closes the alert.